doc/adr/0008-libero-infrastructure.md

```diff
@@ -0,0 +1,34 @@
+# 8. Limit scope of Libero infrastructure
+
+Date: 2020-01-10
+
+## Status
+
+Proposed
+
+## Context
+
+Libero infrastructure (servers, Kubernetes, buckets) supports Libero development by providing demo environments.
+
+Service providers need documentation to learn to run Libero products.
+
+Service providers cater for the disparate, very specific needs of publishers.
+
+Service providers may consolidate their infrastructure with the rest of the platforms.
+
+## Decision
+
+Libero infrastructure should serve two purposes:
+
+- provide *demo* environments to showcase Libero products in certain configurations
+- provide realistic *reference* environments that do not serve real users but can be forked and adapted by service providers to kick start their Libero offering
```

Comment:
Take 'demo' to be something that _is_ running, and 'reference' to be something that is _not_ running.

Reply:
Would only deploy `reference` to make sure it's working, so could be on each build of whatever Infrastructure As Code repository represents it. At the start it probably coincides with `demo` but can see that they may diverge later.